src/Styleguide/Elements/Message.tsx

```diff
@@ -10,14 +10,19 @@ export const StyledFlex = Flex.extend`
 
 interface MessageProps extends FlexProps {
   children: React.ReactNode | null
+  size?: SansSize
 }
 
-export function Message({ children, ...others }: MessageProps) {
+export const Message: SFC<MessageProps> = ({ children, size, ...others }) => {
```

Comment:
@ds300 - `export function Foo` is cleaner IMO but `SFC<Props>` provides slightly better intellisense / typechecking, thats the only reason I updated it.